src/wallet/wallet2.cpp

```diff
@@ -6224,18 +6224,29 @@ void wallet2::store_to(const std::string &path, const epee::wipeable_string &pas
 {
   trim_hashchain();
 
+  const bool had_old_wallet_files = !m_wallet_file.empty();
+  THROW_WALLET_EXCEPTION_IF(path.size() >= 4 && path.substr(path.size() - 4) == ".new",
```

Comment:
Well it could overwrite that file otherwise if it already existed. It's probably not a big deal either way, but might prevent some loss from a buggy script seeing a ".new" file and trying to move it (e.g.)

Reply:
Alright I removed that check